asset/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "@google-cloud/asset-samples",
+  "description": "Samples for the Cloud Asset Client Library for Node.js.",
+  "version": "0.0.1",
+  "license": "Apache-2.0",
+  "author": "Google Inc.",
+  "engines": {
+    "node": ">=8"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/GoogleCloudPlatform/nodejs-docs-samples.git"
+  },
+  "nyc": {
+    "exclude": [
+      "**/*.test.js"
+    ]
+  },
+  "scripts": {
+    "test": "repo-tools test run --cmd ava -- -T 20s --verbose system-test/*.test.js"
+  },
+  "dependencies": {
+    "@google-cloud/pubsub": "*",
+    "@google-cloud/storage": "^2.3.0",
+    "@google-cloud/asset": "^0.1.0",
+    "uuid": "^3.3.2",
+    "yargs": "^12.0.0"
+  },
+  "devDependencies": {
+    "@google-cloud/nodejs-repo-tools": "^3.0.0",
+    "ava": "^0.25.0",
+    "mocha": "^5.2.0",
```

Comment:
mocha, proxyquire, and ava are not used in your tests, so we don't need them. I think we can leave out nyc, too.